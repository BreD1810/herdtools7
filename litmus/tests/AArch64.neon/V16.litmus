AArch64 V16
(* Tests MOVI 64-bit variant *)
{
0:X0=x;0:X1=y;0:X2=z;0:X3=a;
}

 P0                     ;
 MOVI V0.2D,#255        ;
 ST1 {V0.B}[0], [X0]    ;
 ST1 {V0.B}[1], [X1]    ;
 ST1 {V0.B}[8], [X2]    ;
 ST1 {V0.B}[9], [X3]    ;
 forall (x=255 /\ y=0 /\ z=255 /\ a=0)
