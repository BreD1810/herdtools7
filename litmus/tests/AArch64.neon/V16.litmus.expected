Test V16 Required
Histogram (1 states)
1000000:>a=0; x=255; y=0; z=255;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (x=255 /\ y=0 /\ z=255 /\ a=0) is validated
Hash=3ae26a965161c3cc5a1cd800e0198d9b
Observation V16 Always 1000000 0
